x-pack/plugins/security_solution/cypress/e2e/detection_rules/prebuilt_rules_install_update_workflows.cy.ts

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createRuleAssetSavedObject } from '../../helpers/rules';
+import {
+  ELASTIC_RULES_BTN,
+  LOAD_PREBUILT_RULES_BTN,
+  LOAD_PREBUILT_RULES_ON_PAGE_HEADER_BTN,
+  RULES_MANAGEMENT_TABLE,
+  RULES_ROW,
+  TOASTER,
+  UPDATE_PREBUILT_RULES_CALLOUT_BUTTON,
+} from '../../screens/alerts_detection_rules';
+import { waitForRulesTableToBeLoaded } from '../../tasks/alerts_detection_rules';
+import {
+  createNewRuleAsset,
+  installAvailableRules,
+  preventPrebuiltRulesInstallation,
+} from '../../tasks/api_calls/prebuilt_rules';
+import { cleanKibana, resetRulesTableState, deleteAlertsAndRules } from '../../tasks/common';
+import { esArchiverResetKibana } from '../../tasks/es_archiver';
+import { login, visitWithoutDateRange } from '../../tasks/login';
+import { SECURITY_DETECTIONS_RULES_URL } from '../../urls/navigation';
+
+interface Asset {
+  id: string;
+  type: string;
+}
+interface PackageItem {
+  name: string;
+  result: {
+    installSource: string;
+    assets: Asset[];
+  };
+}
+
+describe('Detection rules, Prebuilt Rules Installation and Update workflow', () => {
+  before(() => {
+    cleanKibana();
+    login();
+  });
+  beforeEach(() => {
+    resetRulesTableState();
+    deleteAlertsAndRules();
+    esArchiverResetKibana();
+
+    visitWithoutDateRange(SECURITY_DETECTIONS_RULES_URL);
+  });
+
+  describe('Installation of prebuilt rules package via Fleet', () => {
+    beforeEach(() => {
+      waitForRulesTableToBeLoaded();
+    });
+
+    it('should install package from Fleet in the background', () => {
+      cy.intercept('POST', '/api/fleet/epm/packages/_bulk*').as('installPackage');
+
+      /* Assert that the package in installed from Fleet by checking that
+      /* the installSource is "registry", as opposed to "bundle" */
+      cy.wait('@installPackage').then(({ response }) => {
+        cy.wrap(response?.statusCode).should('eql', 200);
+
+        const packages = response?.body.items.map(({ name, result }: PackageItem) => ({
+          name,
+          installSource: result.installSource,
+        }));
+
+        expect(packages.length).to.have.greaterThan(0);
+        expect(packages).to.deep.include.members([
+          { name: 'security_detection_engine', installSource: 'registry' },
```

Comment:
This is failing as in CI/CD -the Cypress tests- the `security_detection_engine` package is being installed from `bundled` instead of `registry`, as it would be expected.

The FTR config for this test **does not** have the `xpack.fleet.registryUrl` option set to an invalid url, nor `xpack.fleet.developer.bundledPackageLocation` set to some local dir, so Fleet should try to install the package from the registry (the env should have access to EPR).

Currently investigating why this is failing.

Reply:
Solved: https://elastic.slack.com/archives/CNMNXV4RG/p1683033379063079